samples/deployment-requirements.txt

```diff
@@ -0,0 +1,3 @@
+openvino_genai==2024.5.0
```

Comment:
maybe like here 
https://github.com/openvinotoolkit/openvino.genai/blob/1729e78409c514d2f02d833ede179e4d87631c02/pyproject.toml#L18
to support nightly releases?